extensions/amp-bodymovin-animation/0.1/amp-bodymovin-animation.js

```diff
@@ -14,16 +14,27 @@
  * limitations under the License.
  */
 
+import {ActionTrust} from '../../../src/action-trust';
 import {Services} from '../../../src/services';
 import {assertHttpsUrl} from '../../../src/url';
 import {batchFetchJsonFor} from '../../../src/batched-json';
+import {dict} from '../../../src/utils/object';
 import {getIframe, preloadBootstrap} from '../../../src/3p-frame';
 import {isLayoutSizeDefined} from '../../../src/layout';
 import {removeElement} from '../../../src/dom';
 import {user} from '../../../src/log';
 
 const TAG = 'amp-bodymovin-animation';
 
+/** @enum {number} */
+export const PLAYING_STATE = {
```

Comment:
I was hoping calling an action in an invalid state is no-op in lottie player. Can you give it a try? If that's the cause, I recommend not keeping an internal playing state at all. Issues with internal state is they need to get synced with the player which may be unnecessary work and no trivial. For example currently we are already out of sync in autoplay case and also if loop is not infinite and player paused after x loops we stay in playing state.